home/templates/home.html

```diff
@@ -0,0 +1,19 @@
+{% extends "base.html" %}
+
+{% block content %}
+<main>
+  <div class="container-fluid">
+    <ul class="list-unstyled" id="albumList">
+      {% if albumlist %}
+        {% for album in albumlist %}
+          <div class="row">
+            <div class="col-md-12">
```

Comment:
Can this be made better? (use a layout like 3 cols space, 6 cols of content, and 3 cols of space)

Having the whole page contained by this div when you are anyway centering the text doesn't seem right.